packages/cli/openapi-parser/src/openapi/v3/converters/endpoint/convertParameters.ts

```diff
@@ -65,7 +73,7 @@ export function convertParameters({
                           format: undefined,
                           minLength: undefined,
                           maxLength: undefined,
-                          example: getExamplesString(resolvedParameter.example)
+                          example: getExamplesString(resolvedParameter as OpenAPIV3.SchemaObject)
```

Comment:
why do we have to use any casting here?

Reply:
should we just update the signature of the method to accept `OpenAPIV3.SchemaObject | OpenAPIV3.ParameterObject`